app/views/admin/statuses/show.html.haml

```diff
@@ -44,7 +51,52 @@
 
 %hr.spacer/
 
-%h3= t('admin.statuses.history')
+%h3= t('admin.statuses.contents')
+
+.status__card
+  - if @status.reblog?
+    .status__prepend
+      = fa_icon('retweet fw')
+      = t('statuses.boosted_from_html', acct_link: admin_account_inline_link_to(@status.proper.account))
+  .status__content><
+    - if @status.proper.spoiler_text.blank?
+      = prerender_custom_emojis(status_content_format(@status.proper), @status.proper.emojis)
+    - else
+      %details<
+        %summary><
+          %strong> Content warning: #{prerender_custom_emojis(h(@status.proper.spoiler_text), @status.proper.emojis)}
+        = prerender_custom_emojis(status_content_format(@status.proper), @status.proper.emojis)
+
+    - unless @status.proper.ordered_media_attachments.empty?
+      = render partial: 'admin/reports/media_attachments', locals: { status: @status.proper }
+
+    .detailed-status__meta
+      - if @status.application
+        = @status.application.name
+        ·
+      %span.detailed-status__datetime
+        %time.formatted{ datetime: @status.created_at.iso8601, title: l(@status.created_at) }= l(@status.created_at)
+      - if @status.edited?
+        ·
+        %span.detailed-status__datetime
+          = t('statuses.edited_at_html', date: content_tag(:time, l(@status.edited_at), datetime: @status.edited_at.iso8601, title: l(@status.edited_at), class: 'formatted'))
+      - if @status.discarded?
+        ·
+        %span.negative-hint= t('admin.statuses.deleted')
+      - unless @status.reblog?
```

Comment:
I'm not sure on this, I think we'd always want to see the visibility, especially as reblogs can change the visibility?